daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -272,6 +272,7 @@ object Repl {
 
   def prettyDefinitionType(defn: Definition, pkgId: PackageId, modId: ModuleName): String =
     defn match {
+      case DTypeSyn(_, _) => "<type synonym>" // FIXME(JM): pp this
```

Comment:
I am fine not pretty print the type synonyms, but you probably should not the initals of @dajmaki in your comment.

Reply:
removed the JP